bin/rails

```diff
@@ -1,11 +1,4 @@
 #!/usr/bin/env ruby
-# frozen_string_literal: true
-
-begin
-  load File.expand_path('spring', __dir__)
-rescue LoadError => e
-  raise unless e.message.include?('spring')
-end
```

Comment:
What's the reason for removing these bits, and should you also remove bin/spring ?

Reply:
Part of the Rails upgrade. A number of files are replaced automatically during the upgrade so these were over written.